ts/RequestValidator.ts

```diff
@@ -157,28 +165,26 @@ export class RequestValidator {
                 delete validation.disallowExtraFields;
             }
 
-            for (const key of Object.keys(validation)) {
+            for (const key of this.getPrioritizedValidationKeys(validation)) {
                 const paramValidation = RequestValidator.buildValidationParam(validation[key]);
                 if (paramValidation) {
-                    // Check "required" param
                     const type = input ? typeof input[key] : undefined;
-                    if (paramValidation.required === true && (!input || type === 'undefined')) {
```

Comment:
the "required" validation was moved into the method "validateField" (together with all other validations).
I could not find a use case where the repo could benefit from handling this error differently from the rest.